app/views/devise/mailer/unlock_instructions.html.erb

```diff
@@ -1,7 +1,23 @@
-<p>Hello <%= @resource.email %>!</p>
-
-<p>Your account has been locked due to an excessive number of unsuccessful sign in attempts.</p>
-
-<p>Click the link below to unlock your account:</p>
-
-<p><%= link_to 'Unlock my account', unlock_url(@resource, unlock_token: @token) %></p>
+<table class="section">
+  <tr>
+    <td>
+      <table>
+        <tr>
+          <td>
+            <h1 class="section-header">Hello <%= @resource.email %>!</h1>
```

Comment:
lets change `@resource.email` to something else, or just say hello. its kind of strange to be greeted by your email address. Could be something like "Hello! ... Your account (email address) has ben locked..." or whatever makes sense..

*This goes for all instances of `@resource.email` in email views :)